www/docs/user/index.php

```diff
@@ -668,7 +667,6 @@ function display_form ( $details = array(), $errors = array() ) {
     }
 ?>
                 <div class="row">
-                <span class="label">&nbsp;</span>
```

Comment:
There's a couple more further down that could be removed too, for the email updates Yes/No, and the terms/submit button itself.